app/rdf/queries.ts

```diff
@@ -306,175 +301,14 @@ export const getCubeDimensionValuesWithMetadata = async ({
-        cache,
-      }),
-      scaleType === "Ordinal" || scaleType === "Nominal"
-        ? loadResourceLabels({
-            ids: namedNodes,
-            locale,
-            sparqlClient,
-            labelTerm: schema.description,
-            cache,
-          })
-        : [],
-      loadResourceLiterals({
-        ids: namedNodes,
-        sparqlClient,
-        predicates: {
-          identifier:
-            scaleType === "Ordinal" || scaleType === "Nominal"
-              ? {
-                  predicate: schema.identifier,
-                }
-              : null,
-          position:
-            scaleType === "Ordinal"
-              ? {
-                  predicate: schema.position,
-                }
-              : null,
-          color:
-            scaleType === "Nominal" || scaleType === "Ordinal"
-              ? { predicate: schema.color }
-              : null,
-          alternateName: {
-            predicate: schema.alternateName,
-            locale: locale,
-          },
-        },
-        cache,
-      }),
-      dimensionIsVersioned(dimension)
-        ? loadUnversionedResources({ ids: namedNodes, sparqlClient, cache })
-        : [],
-    ]);
-
-    const lookup = new Map(
-      literals.map(({ iri, alternateName, identifier, position, color }) => [
-        iri.value,
-        {
-          alternateName: alternateName
-            ? parseRDFLiteral<string>(alternateName)
-            : undefined,
-          identifier: identifier
-            ? parseRDFLiteral<number>(identifier)
-            : undefined,
-          position: position ? parseRDFLiteral<number>(position) : undefined,
-          color: color ? parseRDFLiteral<string>(color) : undefined,
-        },
-      ])
-    );
-
-    const labelsLookup = new Map(
-      labels.map(({ iri, label }) => [iri.value, label?.value])
-    );
-
-    const descriptionsLookup = new Map(
-      descriptions.map(({ iri, label }) => [iri.value, label?.value])
-    );
-
-    const unversionedLookup = new Map(
-      unversioned.map(({ iri, sameAs }) => [iri.value, sameAs?.value])
-    );
-
-    namedNodes.forEach((iri) => {
-      const lookupValue = lookup.get(iri.value);
-
-      result.push({
-        value: unversionedLookup.get(iri.value) ?? iri.value,
-        label: labelsLookup.get(iri.value) ?? "",
-        description: descriptionsLookup.get(iri.value),
-        position: lookupValue?.position,
-        identifier: lookupValue?.identifier,
-        color: lookupValue?.color,
-        alternateName: lookupValue?.alternateName,
-      });
-    });
-  } else if (literals.length > 0) {
-    literals.forEach((v) => {
-      result.push({
-        value: v.value,
-        label: v.value,
-      });
-    });
-  }
-
-  if (undValues.length > 0) {
-    result.push({
-      value: DIMENSION_VALUE_UNDEFINED, // We use a known string here because actual null does not work as value in UI inputs.
-      label: "–",
-    });
-  }
-
-  return result;
```

Comment:
Nice to be able to remove all this code ! I think it would be cool to have a loadDimensionValues test on a particular dimension where we can be sure that the CONSTRUCT query brings the same result as before.

Should `loadDimensionValues` be renamed `loadDimensionValuesWithMetadata` since I guess we will have places where we do not need the labels etc.. ?